sdks/python/apache_beam/ml/inference/base.py

```diff
@@ -337,6 +337,7 @@ def __init__(self, namespace: str):
     # Metrics
     self._inference_counter = beam.metrics.Metrics.counter(
         namespace, 'num_inferences')
+    self.failed_batches_counter = 0
```

Comment:
This should probably be initialized with `beam.metrics.Metrics.counter(namespace, 'failed_batches_counter')` so that we correctly capture the namespace

Reply:
It looks like this is causing the tests to fail to run FWIW - https://ci-beam.apache.org/job/beam_PreCommit_Python_Commit/25468/consoleFull

Reply:
yes, you should follow the pattern that all the other metrics are using.